.gitlab-ci.yml

```diff
@@ -2319,6 +2338,7 @@ dev_branch_docker_hub-a7-windows:
     - |
       @"
       inv -e docker.publish --signed-push ${SRC_AGENT}:${SRC_TAG}-7-win1809-amd64 datadog/agent-dev:${CI_COMMIT_REF_SLUG}-py3-win1809
+      inv -e docker.publish --signed-push ${SRC_AGENT}:${SRC_TAG}-7-win1809-amd64 datadog/agent-dev:${CI_COMMIT_REF_SLUG}-py3-win1809-jmx
```

Comment:
You need to add -jmx in source part `${SRC_AGENT}:${SRC_TAG}-7-jmx-win1809-amd64` (In all 3 places), otherwise it will just tag the same image twice

Reply:
Oops, good catch

Reply:
Done!

Reply:
I think in the previous step you build:
`-7-jmx-win1809-amd64` not `-7-win1809-amd64-jmx`

Reply:
Damn, I should pay more attention 🤦‍♂

Reply:
I've fixed this and added the jmx version to some places where I think it was missing.

What I find suspicious now is how similar these two lines look:
https://github.com/DataDog/datadog-agent/blob/40737949ff748efa9689483121ee94568d5bc2f5/.gitlab-ci.yml#L2340
https://github.com/DataDog/datadog-agent/blob/40737949ff748efa9689483121ee94568d5bc2f5/.gitlab-ci.yml#L2384

The only difference is that one ends with `-win1809` and the other one with `-1809`. Is this intended?

Reply:
Should be removed, we are using the standard job anymore, won't work anyway

Reply:
Not sure which one should be removed, can you make the change directly on this branch?

Reply:
I assumed these are the ones that had to be removed: https://github.com/DataDog/datadog-agent/pull/5175/commits/b6f3d0e78d6de16755e7898aed63fb80bbffc6f4

Reply:
I supose the `-1809` are the ones that had to be removed. Did it on the last commit of this PR: https://github.com/DataDog/datadog-agent/pull/5175/commits/b6f3d0e78d6de16755e7898aed63fb80bbffc6f4

Reply:
Yep you're totally right!